libr/anal/fcn.c

```diff
@@ -417,6 +417,57 @@ static void queue_case(RAnal *anal, ut64 switch_addr, ut64 case_addr, ut64 id, u
 		switch_addr, id, case_addr);
 }
 
+// TODO: find a better function name
+static int walkthrough_arm_jmptbl_style(RAnal *anal, RAnalFunction *fcn, int depth, ut64 ip, ut64 jmptbl_loc, ut64 sz, ut64 jmptbl_size, ut64 default_case, int ret0) {
+	/*
+	 * Example about arm jump table
+	 *
+	 * 0x000105b4      060050e3       cmp r0, 3
+	 * 0x000105b8      00f18f90       addls pc, pc, r0, lsl 2
+	 * 0x000105bc      0d0000ea       b loc.000105f8
+	 * 0x000105c0      050000ea       b 0x105dc
+	 * 0x000105c4      050000ea       b 0x105e0
+	 * 0x000105c8      060000ea       b 0x105e8
+	 * ; CODE XREF from loc._a_7 (+0x10)
```

Comment:
This example is very useful though I’d remove these codexrefs and replace sym.input labels with their plain addresses